public/js/navigator-js/integration/ViewRecipe.js

```diff
@@ -38,6 +42,15 @@ this.navigatorjs.integration = this.navigatorjs.integration || {};
 		toView: function(viewClass) {
 			this._viewClass = viewClass;
 
+			// This is where the recipe adds a mixin depending on the type of viewClass
+			// added.  This allows for different handling for different 'types' of
+			// view recipes.
+
+			// TODO: Find a more secure way to determine whether class
+			// extends react component
```

Comment:
If you import React and Backbone, you should be able to use `viewClass instanceof React.Component` or `viewClass instanceof Backbone.View` to determine what type of view it is.